ai-creator/AIArtGenerator.js

```diff
@@ -0,0 +1,12 @@
+const deepai = require('deepai');
```

Comment:
Your VsCode Auto formatting doesn't seem to work. 

Can you make sure to install `eslint` and `prettier` extensions in VScode, and make sure your codes are auto formatted correctly?

Reply:
I have then installed I will make sure they are being used correctly!